lib/grape/dsl/parameters.rb

```diff
@@ -165,9 +165,11 @@ def all_or_none_of(*attrs)
       # @raise Grape::Exceptions::UnknownParameter if `attr` has not been
       #   defined in this scope yet
       # @yield a parameter definition DSL
-      def given(attr, &block)
-        fail Grape::Exceptions::UnknownParameter.new(attr) unless declared_param?(attr)
-        new_lateral_scope(dependent_on: attr, &block)
+      def given(*attrs, &block)
+        attrs.each do |attr|
+          fail Grape::Exceptions::UnknownParameter.new(attr) unless declared_param?(attr)
+          new_lateral_scope(dependent_on: attr, &block)
```

Comment:
Hey Ozzy! This is a good start. But I think this is going to open up one scope per each attribute listed and repeat the nested definitions each time. I haven't tested it, so this is a guess, but I believe the behavior it will actually cause is more like if you'd written:

```
given :a do
  requires :c
end
given :b do
  requires :c
end
```

Instead, I think you either want to update how `dependent_on` is parsed to allow *it* to be an array, or create these scopes such that they're nested. That is, emulate:

```
given :a do
  given :b do
    requires :c
  end
end
```